### PR TITLE
Users

### PR DESCRIPTION
**Pull request by mortenwh**
#207 and #208

### CODE REVIEW

**Review by mortenwh — COMMENTED** [review]
At the moment the users are listed in relation to the type of data - I think we should describe more in detail about their level of knowledge and the depth to which they will go in using data and metadata?

For example, as a consumer - what am I looking for? A simple user would like to have a yes/no answer (e.g., "do I need to bring the umbrella?). A more advanced user will need to be able to find and retrieve the raw data used to answer the same question, and use it together with some other data (interoperability).

For the providers, I'm not really sure what we should specify but perhaps something like how much do the providers "interact" or do with the data? Do we talk about people or just things? How are results distributed (e.g., pdf reports, APIs)? How well do they know the consumers of their results?

File: source/users.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
+-----
+Users 
+-----
+
+There are 2 types of users:
+
+1. Those that create / produce data
+2. Those that consume data 
+
+One can both consume data and produce it, or just have one of these roles.
```
Comment (mortenwh) [inline source/users.rst:10]: I have a feeling that if we relate to the table, a user of one level is usually the producer at the next level. The ones calibrating data are users of level-0 and so on. Do we really have a situation where someone only has one role, then?

Comment (lcoram) [reply source/users.rst:10]: Well, I guess it the producer is a sensor... then yes?

Comment (mortenwh) [reply source/users.rst]: yes, I think we do... I specified "at the start/end of the production lines" or something like that

File: source/users.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+           * Merged products based on level 1 data
+    3      * Resampled time series                           * Producer manually resamples time series
+           * Gridded datasets based on level 1 data
+    4      * Weather model data                              * Warnings forecast                               * Climate predictions
+           * Ocean model data                                * Avalanche forecast
+           * Air quality model                               * Nowcast
+           * Climate models                                   
+=========  ================================================  ================================================  ================================================
+
+**In situ observations:**
```
Comment (mortenwh) [inline source/users.rst:47]: Line 47 to 60: I'm not sure what you want to say here... Maybe it can be removed?

Comment (lcoram) [reply source/users.rst:47]: I thought maybe those comments were still relevant? but I suppose they can be deleted if you don't think they are needed.

Comment (lcoram) [reply source/users.rst]: Ok, changed my mind and deleted them...

Comment (mortenwh) [reply source/users.rst]: Ok :) (I've been to the optician, and now lunch..)

**Comment by mortenwh** [comment]
But this should be ok now, shouldn't it? Could you merge and close?

**Comment by lcoram** [comment]
Don't you have to approve it??? since its my merge request...

**Pull request merged**